ArkEcosystem/laravel-fortify
# feat: add recovery code option to the 2fa form

## Summary

- Add a way to add the recovery code on the 2fa form
- Update design of the 2fa form

### How to test

https://app.clickup.com/t/f33r7w

1. Upgrade this package (once merged) on msq
2. Enter as hello@basecode.sh
3. Go to settings > security and enable the 2fa code (you will need a 2fa app)
4. Log out and try to sign again
5. You should see the 2fa form where you can put the 2fa code or you should be able to enter the recovery code.

## Checklist
